LICENSE

```diff
@@ -0,0 +1,121 @@
+Creative Commons Legal Code
+
+CC0 1.0 Universal
```

Comment:
Why not at least CC-BY?

Reply:
from the list on github defaults, it prefers full then cc-by. i've opened another which is better and matches spring-boot.